src/lib.rs

```diff
@@ -150,6 +155,8 @@ impl AstarteSdk {
     async fn subscribe(&self) -> Result<(), AstarteError> {
         let ifaces = self
             .interfaces
+            .read()
+            .await
             .interfaces
             .clone()
```

Comment:
Why are interfaces being cloned here? (Was already there but can be fixed nevertheless) I expect that they just have to be read

Reply:
We can avoid the `clone` borrowing `self.interfaces.interfaces`